packages/sdk-js/README.md

```diff
@@ -18,6 +18,8 @@ Regarding the privacy enhancement, please have a look at our [lightning talk for
 
 ## Documentation
 
+* The Links found here are related to the `master` branch aka the current released version. * Links outside this section point to the version you are currently viewing.
```

Comment:
I would rather use the > tag and optionally make the sentence bold, even thought that might be too much.

Reply:
Ah didn't hat a look at the preview. Is should have been bold. :D That's why there is a `*` in the middle of the sentence. 😁